### ISSUE TITLE
Improve account management

### ISSUE THREAD

**Issue by psiemens** [onflow/flow-cli#31]
### Issue To Be Solved

It's now common for CLI users to switch between multiple accounts when signing and sending transactions. For example, a project may have both an emulator and testnet account with different keys and addresses.

It's currently possible to use multiple accounts by manually editing `flow.json`. Does it make sense to move this to a command? 

```sh
flow signers add <name> <address> ...
```

Maybe we could automatically save accounts in `flow.json` after they are created?

```
flow accounts create --save-as-signer
```

**Comment by sideninja** [comment]
I think this would work nicely with our new config schema. Accounts would be referenced by name and if no account is specified default service account would be selected. I would add an account flag option for any commands that make sense in using different accounts. I think this makes sense to be a flag since it can be used on multiple commands. On the first thought could just be `--account joe` but I would revisit all the commands and see what makes the most sense.

**Comment by sideninja** [comment]
Solved in #81 and #100 the part that is not yet solved is described and still open in #88
